packages/fields-authed-relationship/src/tests/index.test.js

```diff
@@ -0,0 +1,114 @@
+// We don't currently have uniqueness tests for Relationship field types
+import { Text, Password } from '@keystonejs/fields';
+import { PasswordAuthStrategy } from '@keystonejs/auth-password';
+import { multiAdapterRunners, networkedGraphqlRequest, setupServer } from '@keystonejs/test-utils';
+const { createItem } = require('@keystonejs/server-side-graphql-client');
+
+import { AuthedRelationship } from '../index';
+
+function setupKeystone(adapterName) {
+  return setupServer({
+    adapterName,
+    createLists: keystone => {
+      keystone.createList('User', {
+        fields: {
+          username: { type: Text },
+          password: { type: Password },
+        },
+      });
+
+      keystone.createAuthStrategy({
+        type: PasswordAuthStrategy,
+        list: 'User',
+        config: {
+          identityField: 'username',
+          secretField: 'password',
+        },
+      });
+      keystone.createList('Post', {
+        fields: {
+          title: { type: String },
+          // Automatically set to the currently logged in user on create
+          author: {
+            type: AuthedRelationship,
+            ref: 'User',
+          },
+        },
+      });
+    },
+  });
+}
+
+function login(app, username, password) {
+  return networkedGraphqlRequest({
```

Comment:
We're moving away from using `().then(...)`. Can we rewrite this using await instread?

Reply:
Now I think about it, does this need to be a `networkedGraphqlRequest` at all? Can we instead do this with `runCustomQuery`?